tasks/create.yaml

```
@@ -60,7 +60,17 @@ tasks:
     actions:
       - description: "Build the UDS Core Pepr Module"
         cmd: |
-          CUSTOM_PEPR_IMAGE=$( [ "${FLAVOR}" = "registry1" ] && echo "--custom-image ${REGISTRY1_PEPR_IMAGE}" ) || CUSTOM_PEPR_IMAGE=""
+          if [ "${FLAVOR}" = "registry1" ] ; then
+            PEPR_OVERRIDE_IMAGE="--custom-image ${REGISTRY1_PEPR_IMAGE}"
+
+          if [ -n "${PEPR_CUSTOM_IMAGE}" ] ; then
+            # e.g. PEPR_CUSTOM_IMAGE="pepr:dev uds run slim-dev"
+            PEPR_OVERRIDE_IMAGE="--custom-image ${PEPR_CUSTOM_IMAGE}"
+
+          else
+            PEPR_OVERRIDE_IMAGE=""
+          fi
```

```suggestion
          if [ -n "${PEPR_CUSTOM_IMAGE}" ] ; then
            # e.g. PEPR_CUSTOM_IMAGE="pepr:dev uds run slim-dev"
            PEPR_OVERRIDE_IMAGE="--custom-image ${PEPR_CUSTOM_IMAGE}"
          elif [ "${FLAVOR}" = "registry1" ] ; then
            PEPR_OVERRIDE_IMAGE="--custom-image ${REGISTRY1_PEPR_IMAGE}"
          else
            PEPR_OVERRIDE_IMAGE=""
          fi
```

I think this fixes the syntax issues and sets the proper precedence.

---

Might also request that we add `PEPR_OVERRIDE_IMAGE` as a variable to the top of this task, with a description of its usage if possible.

---

Pfft, yeah... was rushed when pushing the update to this -- meant to make those 3 independent if/fi blocks that all just overrode each other top-down -- dunno what I was thinking with that incomplete condition.  Oof.

I think what you have works just fine too though.